frontend/src/components/MissionOverview/OngoingMissionCard.tsx

```diff
@@ -27,13 +30,58 @@ export function OngoingMissionCard({ mission }: MissionProps) {
         navigate(path)
     }
     return (
-        <StyledMissionCard variant="default" style={{ boxShadow: tokens.elevation.raised }} onClick={routeChange}>
-            <Typography variant="h6">INSPECTION</Typography>
+        <StyledMissionCard variant="default" style={{ boxShadow: tokens.elevation.raised }}>
+            <Typography variant="h6" onClick={routeChange}>INSPECTION</Typography>
             <Typography>{mission.name}</Typography>
             <HorisontalContent>
                 <MissionStatusDisplay status={mission.missionStatus} />
                 <MissionProgressDisplay tasks={mission.tasks}/>
+                <MissionControlButtons mission={mission}/>
             </HorisontalContent>
         </StyledMissionCard>
     )
 }
+
+function MissionControlButtons({mission}: MissionProps) {
+    const [status, setStatus] = useState(MissionStatus.Ongoing);
+    enum ControlButton {Pause, Stop, Play}
+    
+    const handleClick = (button: ControlButton) => {
```

Comment:
Perhaps, rather than setting the status immediately in the frontend, this should only ever display the actual status of the mission coming from the robot?
We could implement a loading symbol for when the operator pushes a button while waiting for the backend to confirm that the action has been processed.

Reply:
Probably a good idea

Reply:
Created an issue on it #386